datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp.py

```diff
@@ -0,0 +1,102 @@
+# (C) Datadog, Inc. 2020
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import os
+
+import click
+import yaml
+
+from ..console import CONTEXT_SETTINGS
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='SNMP utilities')
+def snmp():
+    pass
+
+
+def fetch_mib(mib):
+    import pysnmp_mibs
+    from pysmi.codegen import PySnmpCodeGen
+    from pysmi.compiler import MibCompiler
+    from pysmi.parser import SmiStarParser
+    from pysmi.reader import HttpReader
+    from pysmi.writer import PyFileWriter
+
+    target_directory = os.path.dirname(pysnmp_mibs.__file__)
+
+    reader = HttpReader('mibs.snmplabs.com', 80, '/asn1/@mib@')
+    mibCompiler = MibCompiler(SmiStarParser(), PySnmpCodeGen(), PyFileWriter(target_directory))
+
+    mibCompiler.addSources(reader)
+
+    mibCompiler.compile(mib)
+
+
+@snmp.command(context_settings=CONTEXT_SETTINGS, short_help='Translate MIB name to OIDs in SNMP profiles')
+@click.argument('profile')
+@click.pass_context
+def translate_profile(ctx, profile):
```

Comment:
Nice to have.

Could be helpful to have an option `-i/--inplace` that modify the original file (just the `metrics` section)

Reply:
You can't really preserver comments, so I prefer not to offer the option for now.